jsk_fetch_robot/fetcheus/fetch-interface.l

```diff
@@ -42,23 +42,20 @@
      add-new-trajectory-point))
   (:angle-vector-raw (av &optional (tm 3000) &rest args)
    (let* ((prev-av (send self :state :potentio-vector :wait-until-update t))
-          (diff-av (v- av prev-av)))
-     (when (send self :check-continuous-joint-move-over-180 diff-av)
-       (let* (avs (minjerk (instance minjerk-interpolator :init))
-                  (scale-av (send self :sub-angle-vector av prev-av))
-                  dist div)
-         (setq dist (abs (geo::find-extream (coerce diff-av cons) #'abs #'>=)))
-         (setq div (round (/ dist 120.0)))
-         (send minjerk :reset
-               :position-list (list prev-av (v+ prev-av scale-av))
-               :time-list (list tm))
-         (send minjerk :start-interpolation)
-         (send minjerk :pass-time (/ tm div))
-         (dotimes (i div)
-           (setq avs (append avs (list (send minjerk :pass-time (/ tm div))))))
-         (send* self :angle-vector-sequence-raw avs (make-list div :initial-element (/ tm div)) args)
-         (return-from :angle-vector-raw (car (last avs)))))
-     (send-super* :angle-vector av tm args)))
+          (diff-av (v- av prev-av))
+          (minjerk (instance minjerk-interpolator :init))
+          (scale-av (send self :sub-angle-vector av prev-av))
+          avs dist div)
+       (setq dist (abs (geo::find-extream (coerce diff-av cons) #'abs #'>=)))
+       (setq div (max 2 (round (/ dist 120.0))))
+       (send minjerk :reset
+             :position-list (list prev-av (v+ prev-av scale-av))
+             :time-list (list tm))
```

Comment:
I don't see the point of using minjerk interpolator when `div` is 2, since it will simply return `:position-list`.
Seems better just to change Line 61 to `:angle-vector-sequence`, and let the rest unchanged.

Reply:
Checking again appears that minjerk was interpolating the midpoint and final point, instead of the start point and the final one as I thought.

The code was correct the previous way, sorry for misleading comment.